packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Live/Live.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { useSignals, useStore } from 'app/store';
+import { LiveInfo } from './LiveInfo';
+import { LiveButton } from './LiveButton';
+import {
+  Description,
+  WorkspaceInputContainer,
+  WorkspaceSubtitle,
+  ErrorDescription,
+} from '../../elements';
+
+export const Live = observer(() => {
+  const signals = useSignals();
```

Comment:
@Saeris I think you should destructur `signals` & `store`  just like you did in the other files

```js
const { live: { createLiveClicked } } = useSignals();
const {
  editor: { isAllModulesSynced: hasUnsyncedModules },
  live: { isLive, isLoading },
} = useStore();
```